scripts/tests/scripts_regression_tests.py

```diff
@@ -249,7 +249,7 @@ def assert_dashboard_has_build(tester, build_name, expected_count=1):
 
         wget_file = tempfile.mktemp()
 
-        run_cmd_no_fail("wget https://my.cdash.org/api/v1/index.php?project=ACME_test --no-check-certificate -O %s" % wget_file)
+        run_cmd_no_fail("wget https://my.cdash.org/api/v1/index.php?project=ACME_test --no-check-certificate -O %s" % wget_file, timeout=900)
```

Comment:
This one can use SHORT_OP_TIMEOUT.

Reply:
@jgfouca Okay I improved the wget timeout in scripts_regression_tests to use the SHORT_OP_TIMEOUT global constant.